tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -212,6 +212,12 @@ call to find out which methods can be called and which properties it exposes.
 The second is our ``_process`` function, which will work exactly the same
 as the ``_process`` function you're used to in GDScript.
 
+.. note::
```

Comment:
This should be reworded. Since the introduction of runtime classes this would not be always true. Runtime classes do need their own page/section either way. Not sure if it is viable to add this here. If we do add it we should IMHO replace it with a link to that runtime class section when that is written.

Reply:
Thank you for pointing this out, I think I will change my project to runtime class.

When I read this tutorial I thought it to be just like other documentation about scripts, where we assume the reader wants to implement the game functionality, and doing something in the editor is the special case. So I would propose to change the macro in code examples to `GDREGISTER_RUNTIME_CLASS`, and put info that if you want to run your code in the editor, you will need to use `GDREGISTER_CLASS` around the initialize snippet, and in the documentation for `@tool`

Reply:
> So I would propose to change the macro in code examples to `GDREGISTER_RUNTIME_CLASS`, and put info that if you want to run your code in the editor, you will need to use `GDREGISTER_CLASS` around the initialize snippet

I think this makes sense. This example is really a "scripting with GDExtension" example, more than a "extending the engine with GDExtension" example, and in the former most folks probably want runtime classes most of the time.

Reply:
Removed the section, and changed code and screen recording to runtime class.